fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -11860,17 +11860,17 @@
 | `value=(-1)`| (changed)| `"-1"` |
 | `value=(0)`| (changed)| `"0"` |
 | `value=(integer)`| (changed)| `"1"` |
-| `value=(NaN)`| (changed, warning, ssr warning)| `"NaN"` |
+| `value=(NaN)`| (changed, warning)| `"NaN"` |
 | `value=(float)`| (changed)| `"99.99"` |
 | `value=(true)`| (changed)| `"true"` |
 | `value=(false)`| (changed)| `"false"` |
 | `value=(string 'true')`| (changed)| `"true"` |
 | `value=(string 'false')`| (changed)| `"false"` |
 | `value=(string 'on')`| (changed)| `"on"` |
 | `value=(string 'off')`| (changed)| `"off"` |
-| `value=(symbol)`| (changed, error, warning, ssr error)| `` |
-| `value=(function)`| (changed, warning, ssr warning)| `"function f() {}"` |
-| `value=(null)`| (initial, warning, ssr warning)| `<empty string>` |
```

Comment:
Trying to figure this one out. Unit tests suggest that a warning is firing server-side. Does this mean that the attribute table didn't see a SSR warning?

Reply:
It's good when you don't have `ssr warning`. Just `warning` means everything is consistent between client and server.

`ssr warning` confusingly means "client and server match in behavior except warnings". So it's bad to have it.